tests/test_estimators.py

```diff
@@ -118,3 +120,33 @@ def test_estimator_output_type_consistency(output_mode, x_type, estimator):
         ref_type = type(ref_estimator.fit(X, y).predict(X))
 
     assert sknnr_type is ref_type  # noqa: E721
+
+
+@pytest.mark.parametrize("estimator", TEST_YFIT_ESTIMATORS)
+def test_yfit_is_stored(estimator):
+    """Test that y_fit is stored when passed."""
+    X, y = load_moscow_stjoes(return_X_y=True)
+    # Arbitrary split with a fixed constant to prevent zero sum rows
```

Comment:
Too hacky? I couldn't easily find a combo of columns that prevented zero sum row errors, but I also didn't spend a ton of time looking...

Reply:
Not at all.  I think this is totally fine, especially with the comment.